app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
you shouldn add skills, not replace them

Reply:
use self.skills += ["JavaScript", "HTML", "CSS"] in `init` method

Reply:
Hmmm.... I think, I should start from zero point.